float64 steerAngle   		     #Scalar value [-1.0,1.0]
float64 accel                        #It's the throttle, scalar value [0.0, 1.0]
float64 brake
